### PR TITLE
Support integer column_type

### PR DESCRIPTION
**Pull request by mhindery**

### CODE REVIEW

File: tests/integration/test_target_postgres.py [hunk 1]
```diff
@@ -131,17 +132,38 @@ def assert_three_streams_are_into_postgres(self, should_metadata_columns_exist=F
...
         if should_metadata_columns_exist:
             self.assert_metadata_columns_exist(table_one)
             self.assert_metadata_columns_exist(table_two)
             self.assert_metadata_columns_exist(table_three)
+            self.assert_metadata_columns_exist(table_four)
         else:
             self.assert_metadata_columns_not_exist(table_one)
             self.assert_metadata_columns_not_exist(table_two)
             self.assert_metadata_columns_not_exist(table_three)
+            self.assert_metadata_columns_exist(table_four)
```
Comment (koszti) [inline tests/integration/test_target_postgres.py:166]: this should be `assert_metadata_columns_not_exist`

**Comment by koszti** [comment]
In this test we need to change the file name to the one:
https://github.com/transferwise/pipelinewise-target-postgres/blob/5b1bdcacfe8550710631d305fc49635dc31b8212/tests/integration/test_target_postgres.py#L220

**Comment by koszti** [comment]
@mhindery there are some minor problems with the tests, otherwise it looks fine.

Currently I'm working on adding circleci to run the tests and pylint automatically but fixing pylint issues will cause some conflicts in this PR. I merge it now and will fix the test cases as part of the circleci integration.

**Comment by mhindery** [comment]
Perfect, I was a minute to late to push the fixes for the comments, but they are straightforward. Thanks for merging this!

**Pull request merged**